server/game/cards/15-W2D/TheAngler.js

```diff
@@ -0,0 +1,35 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class TheAngler extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Call Out',
+            playType: 'noon',
+            target: {
+                location: 'play area',
+                cardCondition: card => card.locationCard === this.locationCard && card.value === this.value && card.getType() === 'dude' && card.controller !== this.controller
```

Comment:
You can use function to check for the same location, and missing `gameAction`.
```suggestion
                cardCondition: card => this.isInSameLocation(card) && card.value === this.value && card.getType() === 'dude' && card.controller !== this.controller,
                gameAction: 'callout',
```